addons/stock/models/stock_picking.py

```diff
@@ -365,7 +365,7 @@ class Picking(models.Model):
     show_reserved = fields.Boolean(related='picking_type_id.show_reserved')
     show_lots_text = fields.Boolean(compute='_compute_show_lots_text')
     has_tracking = fields.Boolean(compute='_compute_has_tracking')
-    immediate_transfer = fields.Boolean(default=False)
+    immediate_transfer = fields.Boolean(default=False, copy=False)
```

Comment:
why copy=False. If it was an immediate transfer it should be copy as an immediate no?

Reply:
I thought it was just a by-default value set in order to skip some steps: when getting the action, the value is given thanks to the context:
https://github.com/odoo/odoo/blob/472fc28255ad29bcaaa7bd2f68a64018361b2665/addons/stock/models/stock_picking.py#L200-L205
Then, when the user creates the stock picking, it will be automatically confirmed:
https://github.com/odoo/odoo/blob/472fc28255ad29bcaaa7bd2f68a64018361b2665/addons/stock/models/stock_picking.py#L962-L968
(Note: here the `default_immediate_transfer` key is removed but it does not affect the client-side context)

If I want the user to be able to edit the duplicated stock picking, I need the variable set to `False`